application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -378,36 +404,41 @@ record TemporaryData(Instant expiresAt, String duration) {
      * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
      *
      * @param action the action that is being performed, such as banning a user.
-     * @param temporaryData is the message that is being used to display action duration.
-     * @param guild to retrieve the guild.
+     * @param temporaryData if the action is a temporary action, such as a 1 hour mute.
+     * @param guild for which the action was triggered.
      * @param reason for the action.
      * @return the appropriate advice.
      */
-    public static String getDmAdvice(ModerationAction action,
-            @Nullable ModerationUtils.TemporaryData temporaryData, Guild guild, String reason) {
-        if (action == ModerationAction.MUTE) {
-            String durationMessage =
-                    temporaryData == null ? "permanently" : "for " + temporaryData.duration();
-            return """
-                    Hey there, sorry to tell you but unfortunately you have been %s %s in the server %s.
-                    To get in touch with a moderator, you can simply use the **/modmail** command here in this chat. Your message will then be forwarded and a moderator will get back to you soon :thumbsup:
-                    The reason for being %s is: %s
-                    """
-                .formatted(action.getVerb(), durationMessage, guild.getName(), action.getVerb(),
-                        reason);
-        } else if (action == ModerationAction.UNMUTE || action == ModerationAction.UNQUARANTINE) {
-            return """
+    public static RestAction<MessageCreateAction> sendDmAdvice(ModerationAction action,
+            @Nullable ModerationUtils.TemporaryData temporaryData, Guild guild, String reason,
+            PrivateChannel textChannel) {
+        final String COMMAND_NAME = "modmail";
+        if (REVOKE_ACTIONS.contains(action)) {
+            return new CompletedRestAction<>(guild.getJDA(), textChannel.sendMessage("""
                     Hey there, you have been %s in the server %s.
                     This means you can now interact with others in the server again.
                     The reason for being %s is: %s
-                    """.formatted(action.getVerb(), guild.getName(), action.getVerb(), reason);
+                    """.formatted(action.getVerb(), guild.getName(), action.getVerb(), reason)));
```

Comment:
this is incorrect. what u want is just:
```java
return textChannel.sendMessage("blabla");
```
without the `CompoletedRestAction` stuff. that was only needed previously bc u had no rest actions available but just a bare string. but now u have a rest action (from sendMessage)